tools/mo/openvino/tools/mo/ops/split.py

```diff
@@ -154,6 +155,7 @@ def __init__(self, graph: Graph, attrs: dict):
             'version': 'opset1',
 
             'infer': self.infer,
+            'reverse_infer': reverse_bypass_infer,
```

Comment:
please correct. now you set output shape to input. Please create dedicated function. reverse_bypass_infer is not suitable here

Reply:
Agreed and rewritten. Now `reverse_infer` tries get most defined input shapes. If,
```
out_shape_1 = [dyn, 4, 3], out_shape_2 = [7, dyn, 3], axis = 2 
```
then reverse_infer will return source
```
input shape = [7, 4, 6]
```